iothub/service/src/JobClient/JobClient.cs

```diff
@@ -41,6 +47,66 @@ public static JobClient CreateFromConnectionString(string connectionString, Http
             return new HttpJobClient(iotHubConnectionString, transportSettings);
         }
 
+#if !NET451
+
+        /// <summary>
+        /// Creates an instance of <see cref="JobClient"/>.
+        /// </summary>
+        /// <param name="hostName">IoT hub host name.</param>
+        /// <param name="credential">AAD <see cref="TokenCredential"/> to authenticate with IoT hub.</param>
+        /// <param name="transportSettings">The HTTP transport settings.</param>
+        /// <returns>An instance of <see cref="JobClient"/>.</returns>
+        public static JobClient Create(
+            string hostName,
+            TokenCredential credential,
+            HttpTransportSettings transportSettings = default)
+        {
+            if (string.IsNullOrEmpty(hostName))
+            {
+                throw new ArgumentNullException($"{nameof(hostName)} is null or empty");
+            }
+
+            if (credential == null)
+            {
+                throw new ArgumentNullException($"{nameof(credential)} is null");
+            }
+
+            TlsVersions.Instance.SetLegacyAcceptableVersions();
```

Comment:
same, why call this?

Reply:
Will remove. Did not see the implementation was only for NET451